src/adapter/tstype.ts

```diff
@@ -17,6 +17,7 @@ interface AdapterOptions {
     concurrent?: number;
     delay?: number;
     disableLED: boolean;
+    runInconsistent?: boolean;
```

Comment:
Rename to `forceStartWithInconsistentAdapterConfiguration`. Make it long so people see its bad.